src/Workspaces/Core/Portable/Diagnostics/WellKnownDiagnosticCustomTags.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.Diagnostics
+{
+    internal static class WellKnownDiagnosticCustomTags
```

Comment:
Do we want this defined in the analyzer shared project in case it's needed in the future?

Reply:
Not really, we want all our code style analyzers to provide an explicit help link, in which case F1 help is never invoked for them.